tests/generation/test_framework_agnostic.py

```diff
@@ -634,7 +634,7 @@ def test_eos_token_id_int_and_list_beam_search(self):
             "num_beams": 3,
         }
         if is_pt:
```

Comment:
nit: we can actually remove this if/else, if the result is back into being the same :P

Reply:
redundant if/else removed